modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-test/src/testIntegration/java/com/liferay/headless/commerce/admin/channel/resource/v1_0/test/ChannelResourceTest.java

```diff
@@ -283,6 +284,16 @@ private long _getCommerceChannelRelId() throws Exception {
 		return _commerceChannelRel.getCommerceChannelRelId();
 	}
 
+	private AccountEntry _randomAccountEntry() throws Exception {
```

Comment:
call this _addAccountEntry because we are not returning just to object, we are adding it to the system